runtime/sema/type.go

```diff
@@ -3882,6 +3899,28 @@ func (t *CompositeType) MemberMap() *StringMemberOrderedMap {
 	return t.Members
 }
 
+func (t *CompositeType) SupportedEntitlements() (set *EntitlementOrderedSet) {
+	if t.supportedEntitlements != nil {
+		return t.supportedEntitlements
```

Comment:
Use what was checked

```suggestion
	supportedEntitlements := t.supportedEntitlements
	if supportedEntitlements != nil {
		return supportedEntitlements
```